docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -1355,81 +1370,193 @@ intervention. The food vehicle referred to in the table is wheat flour.
-only two data points. Moreover, this coupling seems plausible since the data for
-Kano and Lagos were from the same paper and therefore could have a similar bias.
+and similarly for :math:`b` and :math:`c`.
+
+`Couple <https://en.wikipedia.org/wiki/Coupling_(probability)>`_ the random
+variables :math:`a_\textit{Kano}` and :math:`a_\textit{Lagos}` `comonotonically
+<comonotonicity_>`_, i.e. by giving them the same `percentile rank`_ (and
+similarly for :math:`b` and :math:`c`). This can be done by using `inverse
+transform sampling`_ to generate :math:`a_\textit{Kano}` and
+:math:`a_\textit{Lagos}` from a single uniform random variable :math:`u`. This
+is the same strategy described below for coupling :math:`a`, :math:`b`, and
+:math:`c` for each location, so only one uniform random variable :math:`u` will
+be needed to generate all six subnational numbers for Nigeria, which will then
+be averaged as described above to obtain the national estimates.
+
+The comonotone coupling strategy will create greater uncertainty in the weighted
+average :math:`a_\textit{Nigeria}` than if we sampled the two estimates
+indepdently, and more uncertainty seems like a good idea since we're trying to
+estimate an average for the entire country based on only two data points.
+Moreover, this coupling seems plausible since the data for Kano and Lagos were
+from the same paper and therefore could have a similar bias.
+
+.. todo::
+
+  List the means of a, b, and c for Nigeria, for easier validation later.
+  Also, draw histograms for the distributions.
+
+Joint distribution of :math:`a`, :math:`b`, and :math:`c`
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To ensure that :math:`a\le b\le c` for each location, we will use the
+`comonotone coupling <comonotonicity_>`_ of the three random variables. That is,
+for each location, sample :math:`a`, :math:`b`, and :math:`c` so that they all
+have the same `percentile rank`_ in their respective distributions. This can be
+done by using `inverse transform sampling`_ to generate all three variables
+(:math:`a`, :math:`b`, and :math:`c`) from a single uniform random variable
+:math:`u`.
+
+*Between* countries, the random vectors :math:`(a,b,c)` should be independent.
+That is, the three random vectors :math:`(a,b,c)_\textit{Ethiopia}`,
+:math:`(a,b,c)_\textit{India}`, and :math:`(a,b,c)_\textit{Nigeria}` should be
+mutually independent, where :math:`(a,b,c)_\text{[country_name]}` indicates
+the national estimate  obtained as described above.
+
+.. todo::
+
+  Check whether a, b, and c are actually `stochastically ordered <stochastic
+  ordering_>`_, i.e. their distribution functions satisfy :math:`F_a(x)\ge
+  F_b(x)\ge F_c(x)` for all x. If not, then then the comonotone coupling won't
+  work for guaranteeing :math:`a\le b\le c`, and neither will any other. This
+  could particularly be a problem for Ethiopia since the distribution for
+  :math:`a` has a different form from the others.
```

Comment:
Just a thought, but I wonder if the a_ethiopia distribution is causing us issues if it would be worth it for us to just assume that baseline coverage is just 0 and clearly cite that assumption because my understanding is that the evidence suggests that baseline coverage is pretty much negligible.

Reply:
Yeah, that would be a reasonable approach and would simplify things. I initially included a distribution because we didn't have much data, but if there's data saying it's 0, then we could just go with that.